src/components/sort-bar/SortBar.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class SortBar extends Component {
+  render() {
+    return (
+      <div className="pt-3 pb-3 gray-back">
+        <div className="container">
+          <div className="row mr-3 pr-3">
+            {this.props.info}
+            {this.props.options}
```

Comment:
please, use destructuring here

Reply:
More over this two props can be replaced with one `{children}`:  `<div className="row mr-3 pr-3">{this.props.children}</div>`